buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -184,6 +186,20 @@ class BeamModulePlugin implements Plugin<Project> {
     String tag = null // Sets the image tag (optional).
   }
 
+  // A class defining the configuration for PortableValidatesRunner.
+  class PortableValidatesRunnerConfiguration {
```

Comment:
I feel like it would be better to add the defaults for this configuration object here instead of within the createPortableValidatesRunnerTask definition for clarity to people who are modifying the gradle files. That way they can see the defaults without having to read through the method, and it removes a bunch of the ?: statements in the function definition.